### PR TITLE
Feature/cdh mixed type deprecation

### PR DESCRIPTION
**Pull request by cheinz-sag**
Hi,

can you pls check this PR? This is a 10.15-only change, therefore it is branched from the 10.15 release line.

Thanks
Christoph

### CODE REVIEW

File: content/release-10-15-0/announcements-10-15-0.md [hunk 1]
```diff
@@ -114,7 +114,7 @@ The `SMSResponse` and `SMSResourceReference` events in the `com.apama.cumulocity
 
 #### Implemented
 
-##### Analytics Builder - removal of frgment property
+##### Analytics Builder - removal of fragment property
```
Comment (BeateRixen) [inline content/release-10-15-0/announcements-10-15-0.md:117]: This must be "frgment", see the info in the text.

File: content/release-10-15-0/announcements-10-15-0.md [hunk 2]
```diff
@@ -194,3 +194,15 @@ This is available from the following web page: https://documentation.softwareag.
...
 
 If you need assistance to migrate your streaming analytics logic from Esper to Apama, please contact [Software AG Global Support](/about/contacting-support/).
+
+### Cumulocity IoT DataHub
+
+#### Planned
+
+##### Breaking change in the offloading mechanism - support for mixed types will be deprecated
+
+In a future release of Cumulocity IoT DataHub a newer version of Dremio will be adopted that removes support for mixed data types in a single column. The offloading process will be adapted so that mixed types will be automatically resolved during offloading. Cumulocity IoT DataHub will not be able to read data lake contents containing mixed data types, so corresponding data needs to be converted.
```
Comment (BeateRixen) [inline content/release-10-15-0/announcements-10-15-0.md:204]: Do we need to add a link where conversion of the data is explained? Or is this obvious?

Comment (cheinz-sag) [reply content/release-10-15-0/announcements-10-15-0.md:204]: Good idea, but so far this is not completely clear how that conversion will look like

Comment (BeateRixen) [reply content/release-10-15-0/announcements-10-15-0.md:204]: So perhaps you should add a similar sentence as below?

Comment (cheinz-sag) [reply content/release-10-15-0/announcements-10-15-0.md:204]: We deliberately kept it vague until we have fleshed out more details of that conversion. So I'd tend to keep it like that.

File: content/release-10-15-0/announcements-10-15-0.md [hunk 3]
```diff
@@ -194,3 +194,15 @@ This is available from the following web page: https://documentation.softwareag.
...
+
+##### Breaking change in the offloading mechanism - switch to new table format may introduce incompatibilities
+
+In future releases of Cumulocity IoT DataHub the offloading mechanism may leverage the Apache Iceberg table format, which is an open format supported by Dremio. Due to this change, current offloading configurations may fail, for example due to rarely used data types which are incompatible with the Apache Iceberg format. Further details and remediation actions will be provided closer to the respective release.
```
Comment (MWindel) [inline content/release-10-15-0/announcements-10-15-0.md:208]:
```suggestion
In future releases of Cumulocity IoT DataHub the offloading mechanism may leverage the Apache Iceberg table format, which is an open format supported by Dremio. Due to this change, current offloading configurations may fail, for example, due to rarely used data types which are incompatible with the Apache Iceberg format. Further details and remediation actions will be provided closer to the respective release.
```

Comment (MWindel) [reply content/release-10-15-0/announcements-10-15-0.md:208]: @cheinz-sag we have decided to always use a comma after "for example" (at least in our documentation), but yes it makes the connection clearer to the second part of the sentence.

Comment (cheinz-sag) [reply content/release-10-15-0/announcements-10-15-0.md]: The fact that we are discussing it, indicates that it should be more precise :) Pls see latest commit.

**Review by BeateRixen — APPROVED** [review]

**Review by MWindel — APPROVED** [review]

**Comment by BeateRixen** [comment]
@cheinz-sag Should I merge this or do you need approval from your team?

**Comment by cheinz-sag** [comment]
@BeateRixen The PR can be merged, thanks!

File: content/release-10-15-0/announcements-10-15-0.md [hunk 4]
```diff
@@ -194,3 +194,15 @@ This is available from the following web page: https://documentation.softwareag.
 Further to the CEL (Esper) deprecation notice in [release 10.5](/release-10-5-0/migration-10-5-0/) and the subsequent announcement of end of support in [release 10.7](/release-10-7-0/announcements-10-7-0/) it has been possible to continue using Esper in an unsupported mode. With effect from release 10.15 the CEL (Esper) functionality has been removed completely and cannot be used anymore.
 
 If you need assistance to migrate your streaming analytics logic from Esper to Apama, please contact [Software AG Global Support](/about/contacting-support/).
+
+### Cumulocity IoT DataHub
+
+#### Planned
+
+##### Breaking change in the offloading mechanism - support for mixed types will be deprecated
```
Comment (doernemt) [inline content/release-10-15-0/announcements-10-15-0.md:202]: ##### Breaking change in the offloading mechanism - DEPRECATION OF support for mixed types

(We deprecate with this release and will remove support in future versions)

Comment (cheinz-sag) [reply content/release-10-15-0/announcements-10-15-0.md:202]: Done...

@BeateRixen  Can you pls merge the latest commit? Thanks!

**Comment by cheinz-sag** [comment]
Hi Beate,

sorry for the confusion; simply ignore it...

**Pull request merged**